Doc/whatsnew/3.8.rst

```diff
@@ -67,6 +67,22 @@ Summary -- Release highlights
 New Features
 ============
 
+Assignment expressions
+----------------------
+
+There is new syntax (the "walrus operator", ``:=``) to assign values
+to variables as part of an expression.  Example::
+
+  if (n := len(a)) > 10:
```

Comment:
Writing it this way indicates to the human reader (though not to the interpreter -- see the PEP's fine print about scopes :-) that `n` is only relevant inside the `if` block.

You are really free to write it either way, there is no recommendation intended -- this is just a quick example. (I strongly advise everyone to try and come up with examples that are meaningful, rather than using foo, bar, spam and ham. This one was a compromise. Perhaps someone can come up with a more realistic example where data over (or under!) a certain length must be rejected.